src/Files.Uwp/App.xaml

```diff
@@ -15,8 +15,12 @@
             <!--  Default nav menu item height  -->
             <x:Double x:Key="NavigationViewItemOnLeftMinHeight">32</x:Double>
 
+            <!--  Fix caption buttons background  -->
+            <SolidColorBrush x:Key="WindowCaptionBackground" Color="Transparent" />
+            <SolidColorBrush x:Key="WindowCaptionBackgroundDisabled" Color="Transparent" />
+
             <ResourceDictionary.MergedDictionaries>
-                <XamlControlsResources xmlns="using:Microsoft.UI.Xaml.Controls" ControlsResourcesVersion="Version2" />
+                <XamlControlsResources xmlns="using:Microsoft.UI.Xaml.Controls" />
```

Comment:
Do we still need this reference with WinUI 3?

Reply:
It appears in many win app sdk examples. Haven't tried to remove it.

Reply:
Let's keep it as is 👍